### DISCUSSION TITLE
Any examples on registering a custom protocol

### DISCUSSION THREAD

**Discussion by photino** [DioxusLabs/dioxus#3104] [Q&A]
If I have an element `<a href="myscheme://path_to_handle">Custom</a>`, how can I handle the custom URI? Do we have any examples on registering a custom protocol in `dioxus-desktop`?

**Comment by ealmloff** [comment]
There is an example using a custom asset handler to stream a video here: https://github.com/DioxusLabs/dioxus/blob/main/examples/video_stream.rs#L24-L35
